doc/man3/OSSL_HTTP_parse_url.pod

```diff
@@ -61,6 +78,8 @@ OSSL_HTTP_parse_url(url, ssl, NULL, host, port, NULL, path, NULL, NULL).
 
 =head1 RETURN VALUES
 
+OSSL_HTTP_adapt_proxy() returns a constant proxy hostname string or NULL.
```

Comment:
Maybe mention that the returned pointer is either the `I<proxy>` pointer or pointer to an environment variable or NULL.